build.sbt

```diff
@@ -18,13 +18,14 @@ assemblyOption in assembly ~= {
 
 //Define dependencies.
 libraryDependencies ++= Seq(
-  "org.scalatest"         %% "scalatest"  % "3.0.1" % "test",
-  "org.scalacheck"        %% "scalacheck" % "1.13.4" % "test",
-  "com.typesafe"          % "config"      % "1.3.1",
-  "com.github.pureconfig" %% "pureconfig" % "0.8.0",
-  "com.monovore"          %% "decline"    % "0.3.0",
-  "com.beachape"          %% "enumeratum" % "1.5.12",
-  "org.typelevel"         %% "cats-core"  % "0.9.0"
+  "org.scalatest"         %% "scalatest"             % "3.0.1" % "test",
+  "org.scalacheck"        %% "scalacheck"            % "1.13.4" % "test",
+  "com.typesafe"          % "config"                 % "1.3.1",
+  "com.github.pureconfig" %% "pureconfig"            % "0.8.0",
+  "com.monovore"          %% "decline"               % "0.3.0",
+  "com.beachape"          %% "enumeratum"            % "1.5.12",
+  "org.typelevel"         %% "cats-core"             % "0.9.0",
+  "com.github.pureconfig" %% "pureconfig-enumeratum" % "0.8.0"
```

Comment:
Prefer that pureconfig deps are next to each other. Consider defining a `val PureConfigVersion = "0.8.0"` and using that.